src/commands/api/get.js

```diff
@@ -1,14 +1,21 @@
 const { flags } = require('@oclif/command')
-const { getApiIdentifierArg, reqType, resolvedParam } = require('../../support/command/parse-input')
-const { pipeAsync } = require('../../utils/general')
+const { getApiIdentifierArg, reqType, resolvedParam, splitPathParams } = require('../../support/command/parse-input')
+const { pipeAsync, from } = require('../../utils/general')
 const { getApi } = require('../../requests/api')
 const { getResponseContent } = require('../../support/command/response-handler')
 const BaseCommand = require('../../support/command/base-command')
 
-const versionResponse = content => JSON.parse(content).version
+function versionResponse(content) {
```

Comment:
Just wondering is there benefits to using function over const?

Reply:
Emm it's a bit of an open question really... There are subtle differences - and benefits / drawbacks to both.
What I was trying out was to make all top level functions `es5` to improve the  consistency of files in terms of structure.

When we're using es5 modules i.e (commonJS) everything is set as a variable.
 - imports (require)
 - variables
 - functions

This can make it a little bit messy to read.
Just a thought. I've since reversed this in a following PR

Reply:
Cool, thanks for the explanation :-)